### DISCUSSION TITLE
Security Problem?

### DISCUSSION THREAD

**Discussion by 21hak** [directus/directus#6750] [Archive]
Hi, Directus Team.

I have a question about the security of Directus login. Our team is currently using Directus Cloud Service.
Directus currently sends post api with email and password payload. However, it is easily exposed to all users. There is no settings for something like `Allowed Domains`,  which means anyone using Directus can access my projects and also our admin pages. 
Am I missing somethings or should I create an account for public client?

Thank you!

**Comment by rijkvanzanten** [comment]
I'm not sure what you're talking about.. That being said, for all cloud related questions, use the chatbox on the bottom right of https://directus.cloud. For all potential security problems, please email security@directus.io 👍🏻
